Paws4Me/Paws4Me/UserInterfaces/PetSingleDetailsViewController.swift

```diff
@@ -33,10 +32,12 @@ class PetSingleDetailsViewController: UIViewController {
         UIImage.loadFrom(url: url) { [self] image in
             self.petImageView.layer.cornerRadius = 10
```

Comment:
You can also add this to an extension on `UIImage` 😄

Reply:
Done 👍🏻